kubernetes/resource_kubernetes_service_v1_test.go

```diff
@@ -1104,6 +1147,23 @@ func testAccKubernetesServiceV1Config_loadBalancer_annotations_aws_modified(name
 }
 `, name)
 }
+func testAccKubernetesServiceV1Config_loadBalancer_ipMode(name, ipMode string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_service_v1" "test" {
+  metadata {
+    name = "%s"
+  }
+
+  spec {
+    type = "LoadBalancer"
```

Comment:
this is odd, CI shouldn't be passing here. `ipMode` is never used in this configuration yet the tests seem to be passing. 

Be sure to include the `ip_mode` in the config though I would recommend that we have this config as the first step and have a second step with it updating to `Proxy`, in order to test both the default value being set to "VIP" as well as testing the update case from `VIP` -> `Proxy`

Reply:
This is from the GHA logs:
```
 === CONT  TestAccKubernetesServiceV1_loadBalancer_ipMode
    provider_test.go:249: The Kubernetes endpoint must come from an environment which supports load balancer provisioning for this test to run - skipping
--- SKIP: TestAccKubernetesServiceV1_loadBalancer_ipMode (0.00s)
```

can you look into this? We should definitely fix this so that CI is not skipping the test.